client/scss/components/_forms.scss

```diff
@@ -387,12 +387,6 @@ li.inline:first-child {
   @include transition(opacity 0.2s ease);
 }
 
-.label-uppercase {
-  .field > label {
-    text-transform: uppercase;
-  }
-}
```

Comment:
We should also remove this class from the HTML if we’re removing all the styles here.

Reply:
Thanks! @thibaudcolas 
I'll fix the part you mentioned as well